config.md

```diff
@@ -115,7 +115,6 @@ For POSIX platforms the `mounts` structure has the following fields:
     },
     {
         "destination": "/data",
-        "type": "bind",
```

Comment:
Might as well change this to `"type": "none"` to make this more clear it's not an accidental omission.